bpf/lib/fib.h

```diff
@@ -10,6 +10,12 @@
 #include "common.h"
 #include "l3.h"
 
+/*
+ * Translation into human language: do a fib lookup if there is more than one
+ * output device or per-endpoint routes are enabled (which => !ENABLE_ROUTING)
```

Comment:
Can we get the *why* instead of the *what*? :grin: 
I'm currently missing why your patch needs to change this.

Reply:
@pchaigno I've replaced this with a proper macro

Reply:
+1 to what @pchaigno said - this needs an explanation *why* per-EP routes now always requires a FIB lookup.